action.yml

```diff
@@ -51,15 +55,18 @@ runs:
       shell: bash
 
     - id: create-container
-      run: podman run -dt --privileged $(cat /tmp/mock-container-args) -v ${{ inputs.result-dir }}:/out fedora:latest > action.cid
+      run: podman run -dt --privileged $(cat /tmp/mock-container-args) -v ${{ inputs.result-dir }}:/out ${{ inputs.image }} > action.cid
       shell: bash
 
+    # Skip this job if ${{ inputs.image }} is not the default.  It is expected that any pre-built Docker image
+    # will have mock installed and configured ahead of time.
     - id: init-mock-env
       run: |
         cid=$(cat action.cid)
         podman exec $cid dnf install -y mock
         podman exec $cid mock -r ${{ inputs.chroot }} --init
       shell: bash
+      if: ${{ inputs.image }} == 'fedora:latest'
```

Comment:
There's got to be a way to tie this check to the variable, "inputs.image" in a way that doesn't require hard coding like this.